persistence-modules/hibernate-5-2/pom.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.baeldung</groupId>
+    <artifactId>hibernate-5-2</artifactId>
+    <version>0.1-SNAPSHOT</version>
+    <name>hibernate-5-2</name>
+    <packaging>jar</packaging>
+    <description>Hibernate tutorial illustrating the use of named parameters</description>
+
+    <parent>
+        <groupId>com.baeldung</groupId>
+        <artifactId>parent-modules</artifactId>
+        <version>1.0.0-SNAPSHOT</version>
+        <relativePath>../../</relativePath>
+    </parent>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>5.4.4.Final</version>
```

Comment:
Since we're starting with a brand new module, let's take advantage of the latest releases. This one now is at 5.4.7.Final